astropix.py

```diff
@@ -0,0 +1,701 @@
+"""
+Central module of astropix. This incorporates all of the various modules from the original 
+The class methods of all the other modules are inherited here. 
+
+Author: Autumn Bauman
+"""
+# Needed modules. They all import their own suppourt libraries, 
+# and eventually there will be a list of which ones are needed to run
+from typing import Dict
+from modules.spi import Spi 
+from modules.nexysio import Nexysio
+from modules.decode import Decode
+from modules.injectionboard import Injectionboard
+from modules.voltageboard import Voltageboard
+from bitstring import BitArray
+from tqdm import tqdm
+import pandas as pd
+import regex as re
+import binascii
+import time
```

Comment:
It would be good if you could also update the requirements.txt

Reply:
requirements updated